src/components/Config/Config.tsx

```diff
@@ -1,14 +1,16 @@
 import { useCallback, useMemo, useState } from 'react';
-import { Form, Modal, Button, Checkbox, Typography, Slider, notification } from 'antd';
+import { ethers } from 'ethers';
+import { Form, Modal, Button, Typography, notification, Radio } from 'antd';
 import styled from 'styled-components';
 import { useWallet } from 'use-wallet';
 
 import { CHAIN_ID } from '../../env';
 import { useAuth, useAutomateConnection } from '../../hooks';
 import CopyInput from '../CopyInput';
 import PageTitle from '../PageTitle';
-import { MOBILE_SCREEN_THRESHOLD } from '../../constants';
 
```

Comment:
remove empty line between imports. the exception is an empty line between external imports (things installed via npm) and internal imports (your code). externals should go first, then empty line, then internals, then empty line, then everything else.
also the internal imports should be ordered by distance, i.e. the farther the file is the higher it goes. e.g.
```js
import a from '../../../a'
import b from '../../b'
import c from '../c'
import d from './d'
```
I find this way things are more readable